### PR TITLE
Add Corporate Buzz Words to Entertainment

### PR DESCRIPTION
**Pull request by speedyshreya**
- [Yes] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [Yes] My addition is ordered alphabetically
- [Yes ] My submission has a useful description
- [Yes] The description does not have more than 100 characters
- [Yes ] The description does not end with punctuation
- [Yes ] Each table column is padded with one space on either side
- [Yes] I have searched the repository for any relevant issues or pull requests
- [Yes ] Any category I am creating has the minimum requirement of 3 items
- [Yes] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @speedyshreya ;)

Can you fix the alphabetical order and leave only one whiteline between the categories, please?

**Comment by speedyshreya** [comment]
Hi @matheusfelipeog , thank you for your response. I have made the required changes :)

File: README.md [hunk 1]
```diff
@@ -653,6 +653,7 @@ API | Description | Auth | HTTPS | CORS |
 API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
 | [chucknorris.io](https://api.chucknorris.io) | JSON API for hand curated Chuck Norris jokes | No | Yes | Unknown |
+| [Corporate Buzz Words](https://github.com/sameerkumar18/corporate-bs-generator-api) | REST API for Corporate Buzz Words | No | Yes |Yes |
```
Comment (matheusfelipeog) [inline README.md:656]: each segment must start and end with exactly 1 space

```suggestion
| [Corporate Buzz Words](https://github.com/sameerkumar18/corporate-bs-generator-api) | REST API for Corporate Buzz Words | No | Yes | Yes |
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
@speedyshreya You removed a space in the CORS column, can you fix that, please?

File: README.md (hunk 1 above)
Comment (speedyshreya) [reply README.md]: ah, my bad. Fixed it. Thank you.

**Review by matheusfelipeog — APPROVED** [review]
Thanks for the update @speedyshreya ❤️

**Pull request merged**